SMLHelper/Interfaces/ICraftDataHandler.cs

```diff
@@ -1,9 +1,60 @@
 ﻿namespace SMLHelper.V2.Interfaces
 {
+#if BELOWZERO
+    using System.Collections.Generic;
+#endif
     using Crafting;
 
     public interface ICraftDataHandler
     {
+#if BELOWZERO
```

Comment:
Split `ICraftDataHandler.cs` into multiple files for SN and BZ, this is getting messy.

Reply:
This I will insist that we do not.  
This same interface will define both the Subnautica `CraftDataHandler` and the BelowZero `TechDataHandler`.  

Having a single interface file will reduce the chances of someone forgetting to update one of the two handlers whenever a feature is added or modified.

I will also insist that this particular file isn't messy at all.  
The methods exclusive to each game were organized at the time of the interface and are clearly separated, with the common methods following after.